docs/extensions/guides/renderer-extension.md

```diff
@@ -462,45 +474,64 @@ Consider using the following properties with `updateStatus()`:
 The following shows a very simple implementation of a `ClusterFeature`:
 
 ```typescript
-import { ClusterFeature, Store, K8sApi } from "@k8slens/extensions";
+import { Renderer, Common } from "@k8slens/extensions";
 import * as path from "path";
 
-export class ExampleFeature extends ClusterFeature.Feature {
+const {
+  K8sApi: {
+    ResourceStack, 
+    forCluster, 
+    StorageClass, 
+    Namespace,
+  }
+} = Renderer;
+
+type ResourceStack = Renderer.K8sApi.ResourceStack;
+type Pod = Renderer.K8sApi.Pod;
+type KubernetesCluster = Common.Catalog.KubernetesCluster;
+
+export class ExampleFeature {
+  protected stack: ResourceStack;
 
-  async install(cluster: Store.Cluster): Promise<void> {
+  constructor(protected cluster: KubernetesCluster) {
+    this.stack = new ResourceStack(cluster, this.name);
+  }
 
-    super.applyResources(cluster, path.join(__dirname, "../resources/"));
+  install(config: MetricsConfiguration): Promise<string> {
```

Comment:
Where do `MetricsConfiguration` and `MetricsStatus` come from?

Reply:
Thanks for point these two points out, they should be imported.